number_parser/parser.py

```diff
@@ -252,20 +259,35 @@ def parse_number(input_string, language=None):
     if language is None:
         language = _valid_tokens_by_language(input_string)
 
-    lang_data = LanguageData(language)
+    if numeral_systems is None:
+        if _is_roman(input_string):
+            numeral_systems = ['roman']
+
+        elif language in SUPPORTED_LANGUAGES and not _is_roman(input_string):
+            numeral_systems = ['decimal']
+
+    for numeral_system in numeral_systems:
+        if numeral_system == 'decimal':
+            lang_data = LanguageData(language)
+
+            tokens = _tokenize(input_string, language)
+            normalized_tokens = _normalize_tokens(tokens)
+            for index, token in enumerate(normalized_tokens):
+                if _is_cardinal_token(token, lang_data) or not token.strip():
+                    continue
+                if _is_skip_token(token, lang_data) and index != 0:
+                    continue
+                return None
+            number_built = _build_number(normalized_tokens, lang_data)
+            if len(number_built) == 1:
+                return int(number_built[0])
+            return None
```

Comment:
Those `return None` seem like an issue now within a `for` loop. It looks like they will prevent the `for` loop to reach the next iteration (for the next numeral system).

Maybe you could move this code into a `_parse_decimal` function, and only return if the result is not `None`, else let the `for` loop go to the next numeral system.